docs/platform/what-is-tizen/versions/tizen-5-5-m2.md

```diff
@@ -0,0 +1,71 @@
+ML API sets are based on [NNStreamer](https://github.com/nnsuite/nnstreamer), which allow to construct GStreamer pipelines consisting of conventional GStreamer media plugins and neural network plugins. Although we plan to expand its functionality to on-device training in later releases, ML API sets of the current version of Tizen support machine learning inferences only. In ML API sets, there are two API subsets: Pipeline ML API set and Single ML API set.
+Pipeline ML API set allows you to construct and execute GStreamer pipelines with the conventional GStreamer plugins and neural network plugins provided by NNStreamer. With this API set, you can construct pipelines with multiple instances of neural network models, input streams, and various stream path manipulators.
+
+For input streams, you may use camera, mic, files, or arbitrary inputs from applications. However, your applications need to acquire appropriate privileges for the given inputs. For output streams, you may use files or callback functions of the application.
+Single ML API set allows you to invoke a given neural network model (e.g., .tflite TensorFlow-lite model file) with a single instance of input buffer easily.
+In Tizen 5.5, ML API sets allow TensorFlow-lite 1.13 and custom C libraries for model files. Although, NNStreamer itself supports more neural network frameworks such as TensorFlow, Caffe2, PyTorch, custom Python codes, Movidius NCSDK, and so on. However, device developers can install such frameworks and enable ML API sets to utilize such frameworks.
+
+> **Note**
+>
+> Note that a few of GStreamer plugins that are white-listed are allowed with Tizen Application APIs; on the other hand, all of NNStreamer plugins are allowed.
+
+
+- **Media Vision** :
+Starting with BarCode recognition and generation in Tizen 2.4, features such as detection/recognition/tracking of Face and images have been added to MediaVision. These features are based on traditional computer vision technology and guarantee good execution speed on low-end hardware. With the rapid development of AI technology, the demand for DNN-based vision technologies has increased. In response to this growing trend, Media Vision has applied the AI Framework to some of the existing and new functions.
```

Comment:
Kindly update with: 

In Tizen 2.4 release, bar code recognition and generation features were added to Media Vision. Since then in Tizen 5.5 release,  features such as detection, recognition and tracking of face and images have been added to Media Vision. These features are based on traditional computer vision technology and guarantee good execution speed on low-end hardware. With the rapid development of AI technology, the demand for DNN-based vision technologies has increased. In response to this growing trend, Media Vision has applied the AI Framework to some of the existing and new functions.

Reply:
ok